projects/angular/src/accordion/accordion-panel.ts

```diff
@@ -42,6 +44,11 @@ export class ClrAccordionPanel implements OnInit, OnChanges {
   @Input('clrAccordionPanelDisabled') @HostBinding('class.clr-accordion-panel-disabled') disabled = false;
   @Input('clrAccordionPanelOpen') panelOpen = false;
 
+  /**
+   * Level of the accordion/stepper heading from 1 to 7.
+   */
+  @Input('clrAriaLevel') _ariaLevel: number;
```

Comment:
```suggestion
  @Input('clrAccordionAriaLevel') _ariaLevel: number;
```

Reply:
Yeah, missed the 'Panel' part. We corrected it offline

Reply:
fixed in commit: aa02e3139de0d67e433c4d3fa3ea3cbb7bddb584